src/styles/index.scss

```diff
@@ -1,3 +1,86 @@
-body {
+@import './variables';
+@import './card';
+@import './stars';
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  --active-color: #00acdc;
```

Comment:
CSS custom properties (variables) should be defined in `:root` or in the variables file for better maintainability. Consider moving `--active-color` to the `variables.scss` file.